.github/workflows/run-terraform.yml

```diff
@@ -98,6 +98,12 @@ on:
       arm_tenant_id:
         description: Azure AD Tenant ID
         required: false
+      ts_client_id:
+        description: "The client ID for the Tailscale OAuth application"
+        required: false
+      ts_client_secret:
+        description: "The client secret for the Tailscale OAuth application"
+        required: false
```

Comment:
Should these be under [on.workflow_call.secrets](https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets)?

Reply:
seems like they must be. since this repo is public it can't access the organizational secret, so it must be passed. Is there an alternative solution?

Reply:
yes, works https://github.com/kartverket/skip-apps-terraform/pull/2